cc3d/__init__.py

```diff
@@ -9,7 +9,7 @@
 from os.path import dirname, join, abspath
 from pathlib import Path
 
-__version__ = "4.4.0"
+__version__ = "4.5.0"
```

Comment:
How about 4.4.1, since this is a patch release?

Reply:
Fixed